srcpkgs/ergo/template

```diff
@@ -0,0 +1,40 @@
+# Template file for 'ergo'
+pkgname=ergo
+version=2.13.1
+revision=1
+build_style=go
+go_import_path="github.com/ergochat/ergo"
+short_desc="Modern IRC server (daemon/ircd) written in Go"
```

Comment:
```suggestion
short_desc="Modern IRC server (daemon/ircd)"
```

Reply:
> Modern IRC server (daemon/ircd) with integrated bouncer features and services
Maybe better?

Reply:
sure

Reply:
Drop the `(daemon/ircd)` while you're at it.